### ISSUE TITLE
Start integrating conditional data in the csv into the team object

### ISSUE THREAD

**Issue by BruceJohnJennerLawso** [BruceJohnJennerLawso/scrap#94]
Currently, the only data that is handed off to a game object is data that we absolutely always know will be available, ie SOC for watMu, goals for/against, location, date, result, extra time information, opponent. This is good, because the dataset is complete and consistent going back a full century, but some of the most useful data for evaluating performance (ie shots, powerplay data, time of day) and some less vital, but potentially interesting data (Attendance, length of game, start time, etc.)

Basically, I think the logical next step here should be to have a game method called after tier I where this additional data is available that hands the new data to the game object for use.
